### DISCUSSION TITLE
Hyperparameter Tuning in Spark wtih KerasEstimator

### DISCUSSION THREAD

**Discussion by williambarteck** [horovod/horovod#3188] [Q&A]
Is there any way to combine hyperparameter tuning of a keras model wrapped in the KerasEstimator?

I want to tune model layer values - such as changing the size of a dense layer or dropout rate. I know this is possible outside of spark but I'm hoping I can avoid training the keras model outside of the spark pipeline and retroactively placing it at the end of the pipeline after its trained
